src/main/java/com/team4/testingsystem/controllers/CoachGradeController.java

```diff
@@ -0,0 +1,62 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.dto.CoachGradeDTO;
+import com.team4.testingsystem.services.CoachGradeService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/grades")
+public class CoachGradeController {
+    private final CoachGradeService gradeService;
+
+    @Autowired
+    public CoachGradeController(CoachGradeService gradeService) {
+        this.gradeService = gradeService;
+    }
+
+    @ApiOperation(value = "Get coach's grade for a single question of the test")
+    @ApiResponse(code = 404, message = "Test, question or grade not found")
+    @GetMapping("/{testId}/{questionId}")
+    public CoachGradeDTO getGrade(@PathVariable Long testId, @PathVariable Long questionId) {
+        return new CoachGradeDTO(gradeService.getGrade(testId, questionId));
```

Comment:
Do we really need it?

Reply:
Why not? I mean, it's easier to request a grade for a specific question, instead of parsing array of DTOs.
I was thinking about returning a single Integer here, instead of DTO. Maybe it will be more convenient?

Reply:
What is the usecase for it?

Reply:
When user checks the results, grades for each module are displayed separately.

Reply:
For each module in the same test.
Why can't we send all the results with one request?

Reply:
We can, and we even send...
Ok, I'll remove